packages/desktop-client/src/components/reports/spreadsheets/sheetEngine.ts

```
@@ -0,0 +1,877 @@
+      if (typeof right === 'string' && right.startsWith('Error:')) {
+        throw new Error(right);
+      }
+      switch (node.operator) {
+        case '+':
+          return (left as number) + (right as number);
+        case '-':
+          return (left as number) - (right as number);
+        case '*':
+          return (left as number) * (right as number);
+        case '/':
+          const divisor = right as number;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)